agents/fsevent.go

```diff
@@ -399,11 +405,17 @@ func (fsev FSEvent) V1AuthorizeArgs() (args *sessions.V1AuthorizeArgs) {
 
 // V1InitSessionArgs returns the arguments used in SessionSv1.InitSession
 func (fsev FSEvent) V1InitSessionArgs() (args *sessions.V1InitSessionArgs) {
```

Comment:
please add timezone here as argument to function, so we can use it directly from where we have config.